tests/phpunit/api/v3/CustomValueTest.php

```diff
@@ -230,6 +230,30 @@ public function _testCustomValue($customField, $sqlOps, $type) {
     $params = array('entity_id' => $contactId, 'custom_' . $customId => $selectedValue);
     $this->callAPISuccess('CustomValue', 'create', $params);
 
+    //Test for different return value syntax.
+    $returnValues = [
+      ['return' => "custom_{$customId}"],
+      ['return' => ["custom_{$customId}"]],
+      ["return.custom_{$customId}" => 1],
```

Comment:
Could you please also add 2 more scenarios to this test?

1. Multiple fields to return (array format)
2 Multiple fields to return ('return.custom_x' => 1 format)

Reply:
Updated unit test to cover above scenarios as well 👍